src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -162,7 +178,7 @@ ReportServerSideStats(
     const uint64_t combined_cache_compute_avg_us = AverageDurationInUs(
         stats.cache_hit_time_ns + stats.cache_miss_time_ns +
             total_compute_time_ns,
-        cnt);
+        compute_cnt + cache_hit_cnt);
```

Comment:
@tanmayv25 can you confirm if this new denominator is more accurate than `cnt` as used before (which is success count)?

Reply:
With the new definition of `compute_cnt` it will be fine.

Reply:
Using new definition 👍